suuchi-core/src/main/scala/in/ashwanthkumar/suuchi/store/VersionedStore.scala

```diff
@@ -0,0 +1,87 @@
+package in.ashwanthkumar.suuchi.store
+
+import in.ashwanthkumar.suuchi.store.Store
+import in.ashwanthkumar.suuchi.utils.DateUtils
+
+import scala.util.hashing.MurmurHash3
+
+
+object Versions {
+  def fromString(versionString: String) = versionString.split('|').map(_.toLong)
+  def toString(versions: List[Long]) = versions.map(_.toString).mkString("|")
+}
+
+object VersionedStore {
+  val VERSION_PREFIX = "V_"
```

Comment:
@brewkode Should we convert these `*_PREFIX` to `byte[]` of fixed length instead of a string?